libs/cardano-ledger-binary/testlib/Test/Cardano/Ledger/Binary/Cuddle.hs

```diff
@@ -200,3 +205,18 @@ cddlFailure encoding err =
       , show err
       , "Generated diag: " <> CBOR.prettyHexEnc encoding
       ]
+
+--------------------------------------------------------------------------------
+-- Writing specs to a file
+--------------------------------------------------------------------------------
+
+-- | Write a Huddle specification to a file at the given path
+writeSpec :: Cuddle.Huddle -> FilePath -> IO ()
```

Comment:
Addition of a new function to `cardano-ledger-binary` requires a minor version bump and a changelog entry. So, could you please change the version to `1.3.4.0` and add a lower bound on it in `cardano-ledger-conway`, since that function is being used there.